crates/freeze/src/datasets/erc20_transfers.rs

```diff
@@ -8,7 +8,6 @@ use polars::prelude::*;
 pub struct Erc20Transfers {
     n_rows: u64,
     block_number: Vec<u32>,
-    block_hash: Vec<Option<Vec<u8>>>,
```

Comment:
why delete?

Reply:
This was a mistake, this file should not have been modified